pkg/recommend/advisor/resource_request.go

```diff
@@ -74,19 +79,23 @@ func makeMemConfig(props map[string]string) *config.Config {
 	if !exists {
 		marginFraction = "0.15"
 	}
-
+	targetUtilization, exists := props["resource.mem-target-utilization"]
+	if !exists {
+		targetUtilization = "1.0"
```

Comment:
`
          replicas.min-cpu-target-utilization: "30"
          replicas.max-cpu-target-utilization: "75"
          replicas.cpu-target-utilization: "50"
`

currently xxx-utilization is a 100 percent based float, better to align the same format for new configuration.

also please add it in deploy/craned/deployment.yaml and also the helm chart.